internal/hcserror/helper.go

```diff
@@ -0,0 +1,13 @@
+package hcserror
+
+import "errors"
+
+// IsAny returns true if errors.Is is true for any of the provided errors, errs.
+func IsAny(err error, errs ...error) bool {
```

Comment:
`hcserror` seems to be designed to help handle errors from the HCS. I don't think it's a general package to help with error handling in hcsshim. Given we only use `IsAny` in one package, can we just move this to there and unexport it?